Generator/tests/functional/Hint.test.ts

```diff
@@ -0,0 +1,20 @@
+import { getHint } from "../../../lib/Hint";
+import { TestBoards } from "../testResources";
+import { getBoardArray } from "../../Sudoku";
+import { Solver } from "../../Solver";
+
+describe("get hints", () => {
+    it('get hint works with or without some optional parameters', () => {
```

Comment:
Maybe rename test to 'get hint works without providing strategies parameter'?

Reply:
I named it that since testing without either strategies or solution optional parameters or with having just one of those. Eventually I'll address the TODOs I added and break it up into multiple which actually test functionality of including just strategies and just solution and both.

Reply:
ah, I didn't notice the second 'getHint' call. You're right that is a better test name